Support-and-Services/saas-releases.md

```diff
@@ -6,6 +6,11 @@ For further information please also visit related pages for Exasol SaaS:
 * [SaaS Documentation](https://docs.exasol.com/saas)
 * [Status Page (status.exasol.com)](https://status.exasol.com)
 
+## March 06
+* make jdbc/oracle drivers uploaded with UI available in DB for [connections](https://docs.exasol.com/db/latest/sql/create_connection.htm)
```

Comment:
Perhaps 'JDBC/Oracle drivers uploaded via the file upload feature can now be used as SQL level connections for IMPORT and EXPORT (see ..url...).